### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by Olumayowa-Olowomeye** [mit-acl/android_loomo_ros_core#7]
Hello, I installed the app using the apk but whenever I try to start the app, it almost immediately crashes. There is a brief error where it says "Unfortunately Loomo ROS has stopped working". I'm not really sure where to begin fixing this and any help would be appreciated

**Comment by mfe7** [comment]
Yeah...... I also remember having a tough time debugging with this codebase (but haven't looked at it in a couple years now). Have you confirmed that the network is set up properly? I vaguely remember that it would crash if it couldn't find the ROS master, for instance.

My guess is that this will require some debugging of the source code, rather than installing via apk. There are probably some good ways to pinpoint the exact line that's causing the crash, but to start you might consider adding some print statements throughout the initialization process. @susanni may remember more about this?

**Comment by Olumayowa-Olowomeye** [comment]
I tried to setup the ntp server with the config files you provided but I'm not sure if I did it correctly. I replaced the net.conf file on my ubuntu PC with the ntp_server.conf file but I'm not able to place the ntp_client.conf file on the loomo. I tried using ssh to place it on the loomo but there is no existing ntp.conf and I cant write to it.

**Comment by susanni** [comment]
It's been so long, but I just looked through some of my old notes. I think @mfe7 is right that the app will crash if it can't connect to the ROS master. 

The ROS master has to be up before opening the app. [This comment block](https://github.com/mit-acl/android_loomo_ros_core/blob/master/android_loomo_ros/src/org/ros/android/android_loomo_ros/MainActivity.java#L83-L89) notes that the ROS master URI is being pulled from [params.yaml](https://github.com/mit-acl/android_loomo_ros_core/blob/master/android_loomo_ros/params.yaml) so make sure the yaml is updated with the correct IP address of the ROS master and the [updated yaml file is stored on the Loomo](https://github.com/mit-acl/android_loomo_ros_core/blob/master/android_loomo_ros/params.yaml#L2).

Some extra info from my notes that may be useful:

- roscore requires a static IP address which should be the one in `params.yaml`.
- The [README](https://github.com/mit-acl/android_loomo_ros_core/blob/master/README.md#a-more-streamlined-alternative-to-usb-tethering-using-a-router) describes our networking setup and how we assigned a static IP to each device.
